indico/modules/events/client/js/reviewing/components/UserAvatar.jsx

```diff
@@ -14,13 +14,14 @@ function UserAvatar({user}) {
       className="i-timeline-item-label avatar-placeholder"
       style={{backgroundColor: user.avatarBgColor}}
     >
-      {user.fullName[0]}
+      {user.anonymous ? '' : user.fullName[0]}
```

Comment:
We could check if the `id` is `-1` and probably skip having an anonymous attribute 
(as it seems to be just for the avatar)

Reply:
I had this initially; but I preferred being more explicit in the end. The only reason I'm sending the `-1` id (instead of `null`) is to make PropTypes happy without finding every single place where we're using it or passing it along ;)